### PR TITLE
Fix after release 5.7.23

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: mysql-test/t/percona_log_slow_innodb.test [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#
```
Comment (laurynas-biveinis) [inline mysql-test/t/percona_log_slow_innodb.test:1]: Please remove the whole file not just the first line

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Please refer to which release in the commit message

File: mysql-test/t/percona_log_slow_innodb.test (hunk 1 above)
Comment (EvgeniyPatlan) [reply mysql-test/t/percona_log_slow_innodb.test:1]: @laurynas-biveinis done

**Comment by EvgeniyPatlan** [comment]
commit message has been updated

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**